include/LightGBM/feature_group.h

```diff
@@ -181,6 +185,8 @@ class FeatureGroup {
   std::unique_ptr<Bin> bin_data_;
   /*! \brief True if this feature is sparse */
   bool is_sparse_;
+  /*! \brief Threshold for treating a feature as a sparse feature */
+  double sparse_threshold_;
```

Comment:
I think the ```is_enable_sparse``` you said is the ```is_sparse``` ? Not all ```feature_group``` are sparse, so put it into feature_group is reasonable. But all feature_group share the same sparse_threshold. 
As for the ```is_enable_sparse```, it is a member in the config class, not be saved in the feature_group.